cpu/native/startup.c

```diff
@@ -133,6 +134,9 @@ static const struct option long_opts[] = {
 #ifdef MODULE_SOCKET_ZEP
     { "zep", required_argument, NULL, 'z' },
 #endif
+#ifdef MODULE_NATIVE_EUI_PROVIDER
+    { "eui64", required_argument, NULL, 'Z' },
```

Comment:
Why is it required? From what I see it could be optional.

Reply:
This only means that the `--eui64` parameter requires an argument :wink:

Reply:
Sorry, nvm